sessions/react-usestate-and-events-advanced/lifting-up-state-personal-details/src/Form.js

```diff
@@ -0,0 +1,26 @@
+import {useState} from 'react';
+
+export default function Form({onUserCreate}) {
+	const [name, setName] = useState('');
+	const [email, setEmail] = useState('');
+
+	function handleSubmit(event) {
+		event.preventDefault();
+		const form = event.target;
+		const {name, email} = form.elements;
+		setName(name.value);
+		setEmail(email.value);
+		form.reset();
+	}
+
+	return (
+		<form aria-labelledby="user" onSubmit={handleSubmit}>
+			<h2 id="user">Form with useState</h2>
+			<label htmlFor="name">Name: </label>
+			<input id="name" type="text" placeholder="John Doe" />
+			<label htmlFor="email">Email: </label>
+			<input id="email" type="email" placeholder="john@doe.com" />
```

Comment:
Rename input fields and add `name` attribute

```suggestion
			<label htmlFor="userName">Name: </label>
			<input id="userName" name="userName" type="text" placeholder="John Doe" />
			<label htmlFor="userEmail">Email: </label>
			<input id="userEmail" name="userEmail" type="email" placeholder="john@doe.com" />
```